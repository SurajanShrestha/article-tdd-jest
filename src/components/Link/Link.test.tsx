import renderer from "react-test-renderer";
import { Link } from ".";

// Or you could use it() instead of test()
// it() is an alias of test()
test("link renders correctly", () => {
  const tree = renderer
    .create(<Link page="http://www.instagram.com">Instagram</Link>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
