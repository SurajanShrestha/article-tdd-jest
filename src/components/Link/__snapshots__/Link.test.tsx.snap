// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`link renders correctly 1`] = `
<a
  className="normal"
  href="http://www.instagram.com"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  Instagram
</a>
`;
